### PR TITLE
Move AutobahnTestsuite to extra module

### PR DESCRIPTION
**Pull request by normanmaurer**
Motivation:
We should move the AutobahnTestsuite to an extra module. This allows easier to run only the testsuite or only the autobahntestsuite

Modifications:

Create a new module (testsuite-autobahn)

Result:

Better project structure.

### CODE REVIEW

**Comment by Scottmitch** [comment]
Build failure attributed to https://github.com/netty/netty/issues/5850#issuecomment-281158299

File: testsuite-autobahn/pom.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>netty-codec-http</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+  </dependencies>
+
+
```
Comment (Scottmitch) [inline testsuite-autobahn/pom.xml]: nit: kill line

**Comment by normanmaurer** [comment]
Triggered a build again

File: testsuite-autobahn/pom.xml (hunk 1 above)
Comment (normanmaurer) [inline testsuite-autobahn/pom.xml]: sure... anything else or just this ? :)

Comment (Scottmitch) [inline testsuite-autobahn/pom.xml]: just this 1 nit ... ship it when ready

File: testsuite-autobahn/pom.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      </properties>
+    </profile>
+  </profiles>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>me.normanmaurer.maven.autobahntestsuite</groupId>
+        <artifactId>autobahntestsuite-maven-plugin</artifactId>
+        <version>0.1.3</version>
```
Comment (md-5) [inline testsuite-autobahn/pom.xml]: Old version :)

Comment (normanmaurer) [inline testsuite-autobahn/pom.xml]: yep... will be fixed when merge this one

**Comment by normanmaurer** [comment]
Cherry-picked into 4.1 (2a72c876734abfdfa1fe12041b7d3f0e808cc4ab) and 4.0 (866107a)

**Pull request closed without merging**